lib/rules/font-weight-notation/index.mjs

```diff
@@ -84,26 +82,24 @@ const rule = (primary, secondaryOptions, context) => {
 						continue; // Not `normal` for font-weight
 					}
 
-					if (checkWeight(decl, valueNode)) {
+					if (checkWeight(decl, valueNode, parsedValue)) {
 						break; // Stop traverse if font-weight is processed
 					}
 				}
 
-				checkWeight(decl, valueNode);
-			}
-
-			if (context.fix) {
-				// Autofix after the loop ends can prevent value nodes from changing their positions during the loop.
```

Comment:
We don't have a test case that failed.
@ybiquitous do you have one?

Reply:
No. I guess there is no actual problem, so let's go on.